libs/common/src/lib/prompt-lab/prompt-lab.component.ts

```diff
@@ -61,15 +63,126 @@ export class PromptLabComponent implements OnInit {
   currentQuery = '';
   currentPrompt = '';
   queries: string[] = [];
-  prompts: string[] = [];
 
   configBackup?: { [id: string]: any };
   learningModels?: LearningConfiguration;
   loadingModal?: ModalRef;
   progress$ = new BehaviorSubject<number | null>(null);
-  results: { query: string; data: { prompt?: string; results: { model: string; answer: string }[] }[] }[] = [];
-  hasResults = new BehaviorSubject(false);
-  modelCollapsed: { [id: string]: boolean } = {};
+  results: { query: string; data: { prompt?: string; results: { model: string; answer: string }[] }[] }[] = [
```

Comment:
why do we need hard-coded results?

Reply:
we don't, I just forget to remove them 🙄